lib/auth/helpers.go

```diff
@@ -840,14 +867,27 @@ func (t *TestTLSServer) Stop() error {
 
 // NewServerIdentity generates new server identity, used in tests
 func NewServerIdentity(clt *Server, hostID string, role types.SystemRole) (*Identity, error) {
+	private, public, err := clt.GenerateKeyPair("")
```

Comment:
Suggestion: for consistency?

```suggestion
	priv, pub, err := clt.GenerateKeyPair("")
```